kvdb/kvdb_bolt.go

```diff
@@ -0,0 +1,23 @@
+// +build !kvdb_etcd,!js
+
+package kvdb
+
+// TestBackend is a constant that allows the running backend to be
+// identified in tests.
+const TestBackend = BoltBackendName
```

Comment:
For the non-test code I think we shouldn't make assumptions on what build tags should or should not be used in what combinations within the code. This approach will lead to errors if one tries to build the `kvdb` package with both the `kvdb_postgres` and `kvdb_etcd` modules turned on.
So maybe now is the time to move the content of this file (and as a dependency `channeldb.MakeTestDB()`) into test-only files?

Reply:
I agree, but this also would imply refactoring all our tests to run with postgres/etcd/bbolt as needed, as now this is done through the build tag hack.

Reply:
Reverted the backend initialization code and used the same logic that is used for etcd.